deps/osxunwind.def

```diff
@@ -0,0 +1,2 @@
+-nr:___register_frame:___jl_register_frame
+-nr:___deregister_frame:___jl_deregister_frame
```

Comment:
this PR will also be a bit inconvenient for everyone, since there is no automated rebuild being triggered by the makefile. i'm also mildly concerned that this patch would be confusing to packagers since it is causing action-at-a-distance inside the libosxunwind.dylib via deps/Makefile